packages/client/src/internals/executeChain.ts

```diff
@@ -17,7 +17,12 @@ export function executeChain<
   op: Operation<TInput>;
 }) {
   type TError = TRPCClientError<TRouter>;
-  const $result = observableSubject<TOutput | null, TError>(null);
+  const $result = observableSubject<
+    { type: 'init' } | { type: 'data'; data: TOutput },
+    TError
+  >({
+    type: 'init',
+  });
```

Comment:
This was the culprit - when we initialize as call we started with an `observableSubject` of `null` and we also used this to distinguish if a request was pending or had results.

Reply:
I get why using `null` is not possible and I think you have found a working solution. I wonder if it's possible to make it a little more intuitive. We cannot use `undefined` instead of `null`, right?